src/extension.ts

```diff
@@ -127,6 +138,73 @@ export function activate(context: ExtensionContext) {
         languageClient.sendRequest(ExecuteCommandRequest.type, request);
     }));
 
+    // Create the "open hub" command
+    // This is activated from the status bar (see below)
+    context.subscriptions.push(
+    commands.registerCommand("sourcery.hub.open", () => {
+
+      // Command is handled by the language server
+      languageClient
+        .sendRequest(ExecuteCommandRequest.type, {
+          command: "sourcery.openHub",
+          arguments: [],
+        });
+
+      // Open a webview panel and fill it with a static empty page
+      // The iframe handles loading the actual content
+      const panel = window.createWebviewPanel(
+        "sourceryhub",
+        "Sourcery Hub",
+        ViewColumn.Active,
+          {
+            enableScripts: true,
+          }
+      );
+      panel.webview.html = `<!DOCTYPE html>
+        <head>
```

Comment:
<!-- Polish comments are like a suggestion, where there is nothing necessarily wrong with the relevant content, there's just some ways to immediately improve the quality. -->
**💄 polish** ‏Include a Sourcery favicon

Reply:
Done - I don't think this has any effect in VSCode though.

Reply:
Ahh, maybe there's another mechanism? Tabnine have a favicon like image in their hub